orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartWaitingExecutionsHandler.kt

```diff
@@ -51,11 +51,10 @@ class StartWaitingExecutionsHandler(
             ?.let { newest ->
               log.info("Starting queued {} {} {}", newest.application, newest.name, newest.id)
               queue.push(StartExecution(newest))
-              (pipelines - newest)
```

Comment:
Something is evidently wrong with this when running real pipelines but works okay in the test.